arkane/statmech.py

```diff
@@ -200,15 +200,15 @@ def load(self, pdep=False):
         species object.
         """
         path = self.path
-        TS = isinstance(self.species, TransitionState)
+        is_ts = isinstance(self.species, TransitionState)
         _, file_extension = os.path.splitext(path)
         if file_extension in ['.yml', '.yaml']:
-            if TS:
```

Comment:
Was this line deleted intentionally?

Reply:
I think the PR replaces it with `is_ts`, but probably this commit only captured the deletion. I'll sort the commits out